internal/observability/event/node.go

```diff
@@ -11,17 +11,17 @@ import (
 	"github.com/hashicorp/go-bexpr"
 )
 
-// Node represents an eventlogger.Node which filters events based on allow and
+// cloudEventsFormatterFilter represents an eventlogger.cloudEventsFormatterFilter which filters events based on allow and
```

Comment:
Does it make sense to rename the file based on the renaming inside it? Just for organizational purposes?

Reply:
ty!